### DISCUSSION TITLE
Réduction des altérations d'état dans le jeu

### DISCUSSION THREAD

**Discussion by Ravenclaw-l** [Crownicles/Crownicles#2335] [Nouvelles idées]
_Désolée si ce message est redondant avec un autre, mais je n'ai pas repéré de sujet similaire en cours._

**Est-il possible de réduire le nombre d'altérations d'état dans le jeu ?**

Actuellement, la fréquence des alté rend l'expérience de jeu un peu déroutante, autant pour les nouveaux joueurs que pour les anciens.

Je vous laisse avec ces magnifiques screens de mon jeu ce matin.

![alté1](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/149114595/b37e8ebd-b8a6-4983-aca0-c388d4508de6)
![alté2](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/149114595/f3b689a4-4030-4b28-9fdc-8f7d1e62dbd8)
![alté3](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/149114595/cacc681a-9ff0-41e3-b15e-f7ca3197056b)

**Comment by BastLast** [comment]
C'est un problème que je veux régler avec l'issue des jetons :)

**Comment by Procion06** [comment]
une matinée comme une autre j'ai envie de dire xD
il y a des altés partout et tout le temps (events, mini-events)
et comme ce n'était pas suffisant ils ont eu la bonne idée de rajouter les gobelets, histoire d'être bien sûr que tu ne fasses pas trop de rapports à la suite sans te manger une bonne alté qui fait plaisir ^^

je pense que beaucoup de joueurs abandonnent le jeu à cause de ça

**Comment by Ravenclaw-l** [comment]
En cumulant toutes mes altérations de la journée (mini-event et event ) j'en suis à 32h35.
0h30(mini-event) + 5h00(event) + 1h10(mini-event) + 11h00(event) + 1h25(mini-event) + 2h00(mini-event) + 0h30(mini-event) + 11h00(mini-event) 

Edit : 33h35, je viens de reprendre 1h
